fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckFragment.kt

```diff
@@ -122,6 +123,9 @@ class ReviewQualityCheckFragment : BottomSheetDialogFragment() {
                 if (bottomSheetState == BottomSheetViewState.FULL_VIEW) {
                     behavior?.state = BottomSheetBehavior.STATE_EXPANDED
                 }
+                if (requireContext().isScreenReaderEnabled) {
```

Comment:
I think we should pass `isScreenReaderEnabled` boolean as a constructor param to `ReviewQualityCheckBottomSheetStateFeature`. So this check is done there and the callback returns `BottomSheetViewState.FULL_VIEW`. Then the callback will map `FULL_VIEW` to behaviour of opening the sheet fully. This will also allow this logic to be tested in `ReviewQualityCheckBottomSheetStateFeature`.